src/controllers/urls.go

```diff
@@ -2,16 +2,17 @@ package controllers
 
 import (
 	"errors"
-	"github.com/gofiber/fiber/v2"
-	"github.com/samber/lo"
-	"gorm.io/gorm"
 	"math"
 	"math/rand"
 	"onepixel_backend/src/db"
 	"onepixel_backend/src/db/models"
 	"onepixel_backend/src/utils"
 	"onepixel_backend/src/utils/applogger"
 	"sync"
+
```

Comment:
why the juggling of imports? needed?

Reply:
I tried to split the code to improve readability. I'll check again to remove any unused imports. If you prefer, I can reduce the changes and group them together more closely.

Reply:
ah ok that's fine then